tests/pkg/mcufont/tests/01-run.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2023 Koen Zandberg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    # put here the pexpect code that checks the output of the test application.
+    pass
```

Comment:
This does nothing but could be used to compare the message printed to stdout (see tests/pkg/qrcode_generator)

Reply:
I've added an output comparison in a similar style as the qrcode_generator test